css/docs.scss

```diff
@@ -0,0 +1,432 @@
+---
+---
```

Comment:
can you show the output of broken page that was in the issue

Reply:
Yeah, I've updated  the issue comment